vice.go

```diff
@@ -151,6 +151,63 @@ func (e *ExposerApp) UpsertDeployment(job *model.Job) error {
 	return nil
 }
 
+func (e *ExposerApp) countJobsForUser(username string) (int, error) {
+	set := labels.Set(map[string]string{
+		"username": username,
+	})
+
+	listoptions := metav1.ListOptions{
+		LabelSelector: set.AsSelector().String(),
+	}
+
+	podlist, err := e.clientset.CoreV1().Pods(e.viceNamespace).List(listoptions)
```

Comment:
I would recommend counting the number of deployments here rather than the number of pods. At some point we may allow multiple replicas for each deployment for load-balancing or fail-over, which would throw this count off.

Reply:
Ah, that's a good point. I'll update the code.